packages/react-core/src/demos/Animations/examples/Animations.tsx

```diff
@@ -0,0 +1,1699 @@
+import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
+import QuestionCircleIcon from '@patternfly/react-icons/dist/esm/icons/question-circle-icon';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-icon';
+import PowerOffIcon from '@patternfly/react-icons/dist/esm/icons/power-off-icon';
+import PortIcon from '@patternfly/react-icons/dist/esm/icons/port-icon';
+import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
+import AutomationIcon from '@patternfly/react-icons/dist/esm/icons/automation-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import imgAvatar from '@patternfly/react-core/src/components/assets/avatarImg.svg';
+import SearchIcon from '@patternfly/react-icons/dist/esm/icons/search-icon';
+import pfLogo from '@patternfly/react-core/src/demos/assets/pf-logo.PF-HorizontalLogo-Color.svg';
+import MultiContentCard from '@patternfly/react-component-groups/dist/dynamic/MultiContentCard';
+import { applicationsData } from './ResourceTableData';
+import SkeletonTable from '@patternfly/react-component-groups/dist/dynamic/SkeletonTable';
+
+export const Animations: FunctionComponent = () => {
+  const drawerRef = useRef<HTMLElement | null>(null);
+
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [isKebabDropdownOpen, setIsKebabDropdownOpen] = useState(false);
+  const [isDrawerExpanded, setIsDrawerExpanded] = useState(false);
+  const [isAlertVisible, setIsAlertVisible] = useState(false);
+  const [isAlert2Visible, setIsAlert2Visible] = useState(false);
+  const [isAlert3Visible, setIsAlert3Visible] = useState(false);
+  const [showForm, setShowForm] = useState(false);
+  const [alert1Id, setAlert1Id] = useState('');
+  const [alert2Id, setAlert2Id] = useState('');
+  const [alert3Id, setAlert3Id] = useState('');
+
+  interface UnreadMap {
+    [notificationId: string]: boolean;
+  }
+
+  const [activeItem, setActiveItem] = useState<number | string>(0);
+  const [activeGroup, setActiveGroup] = useState<string | null>(null);
+  const [isUnreadMap, setIsUnreadMap] = useState<UnreadMap | null>({
+    'notification-1': true,
+    'notification-2': true,
+    'notification-3': false,
+    'notification-4': false
+  });
+
+  const [shouldShowNotifications, setShouldShowNotifications] = useState(true);
+
+  interface ActionsMenu {
+    [toggleId: string]: boolean;
+  }
+
+  const handleShowForm = () => {
+    setShowForm(!showForm);
+  };
+
+  const [isActionsMenuOpen, setIsActionsMenuOpen] = useState<ActionsMenu | {}>({});
+
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  const [shouldNotify, setShouldNotify] = useState(false);
+  const prevUnreadCountRef = useRef(0);
+
+  const getNumberUnread = () => {
```

Comment:
You'll need to put this into a `useCallback` to prevent it from causing infinite rerenders since it's listed as a dep in some of your `useEffect`s